Makefile

```diff
@@ -195,28 +195,32 @@ example-dashboards:
 # We use rfratto/seego for building these cross-platform images. seego provides
 # a docker image with gcc toolchains for all of these platforms.
 dist: dist-agent dist-agentctl
-	for i in dist/*; do zip -j -m $$i.zip $$i; done
 	make dist-packages
+	for i in dist/agent*; do zip -j -m $$i.zip $$i; done
```

Comment:
@jdbaldry heads up, I swapped the order of these and changed it to only zip files starting in `agent`. The way it worked before was causing both the zipped and non-zipped version of the linux-amd64 binaries to be added as release assets